.github/workflows/test-python-publish.yml

```diff
@@ -0,0 +1,46 @@
+# This workflows will upload a Python Package to Test PyPI
+# For more information see: https://github.com/marketplace/actions/pypi-publish
+
+name: TEST Upload Python Package
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
If there are some branches, some test versions will be released, right?

If so, it's enough when the codes will be merged to main, I think. What do you think?

Reply:
Purpose of this is to test if the package can be built and pushed to test pypi before running publish job.

> If so, it's enough when the codes will be merged to main, I think. What do you think?
Probably you mean running this job only for `push: branches: [main]` is enough, correct?

Reply:
Got it.
I don't have a strong opinion about it and we don't use test package now.
 so it looks problem.